src/test/java/seedu/address/ui/CommandHistoryTest.java

```diff
@@ -45,9 +33,9 @@ public void getPrevious_completeHistory_isMatching() {
         commandHistory.add("1");
         commandHistory.add("2");
         commandHistory.add("3");
-        assertAll(() -> assertEquals("3", commandHistory.getPrevious()), () ->
-                        assertEquals("2", commandHistory.getPrevious()), () ->
-                assertEquals("1", commandHistory.getPrevious()));
+        assertAll(() -> assertEquals("3", commandHistory.getPrevious("")), () ->
+                        assertEquals("2", commandHistory.getPrevious("")), () ->
+                    assertEquals("1", commandHistory.getPrevious("")));
```

Comment:
my suggested change was only 4 spaces, my bad. It was my intention to align all the assertEquals rather than just add 4 spaces lmao.